src/python/grpcio_observability/grpc_observability/server_call_tracer.cc

```diff
@@ -226,7 +226,7 @@ void PythonOpenCensusServerCallTracer::RecordEnd(
   if (PythonCensusStatsEnabled()) {
     const uint64_t request_size = GetOutgoingDataSize(final_info);
     const uint64_t response_size = GetIncomingDataSize(final_info);
-    double elapsed_time_ms = absl::ToDoubleMilliseconds(elapsed_time_);
+    double elapsed_time_ms = absl::ToDoubleSeconds(elapsed_time_);
```

Comment:
Seems like there's now a mismatch between the variable name and the unit used? Is this double supposed to hold seconds or milliseconds?